NEWS.md

```diff
@@ -1,3 +1,7 @@
+# gridtext (development version)
+
+- Transition to curl package and drop RCurl dependency
```

Comment:
This needs to be moved under the "gridtext 0.1.4.9000" heading, as that is the development version, and it should replace "No changes yet."